### PR TITLE
Added account settings section where a user can swap the send/new lin…

### PR DESCRIPTION
**Pull request by amaddio**
…e behaviour. By experience I have noticed that most of the users tend to like the opposite behaviour.

### CODE REVIEW

**Comment by jasonblais** [comment]
Swapping Eric for PM review since he owns end user features.

File: source/help/getting-started/messaging-basics.rst [hunk 1]
```diff
@@ -3,9 +3,15 @@ Messaging Basics
...
 --------------
 
+.. |Main menu icon| image:: ../../images/sidebar_main_menu_icon.png
+  :width: 33
+  :alt: Main menu
+
 **Write messages** using the text input box at the bottom of the screen.
 Press ENTER to send a message. Use SHIFT+ENTER to create a new
-line without sending a message.
+line without sending a message. To swap send/new line keyboard behavior
```
Comment (esethna) [inline source/help/getting-started/messaging-basics.rst]: Propose: 
```
To send messages on CTRL+ENTER and use ENTER to insert new lines go to **Main Menu** > **Account Settings** > **Advanced** > **Sent messages on CTRL + ENTER**
```

File: source/help/getting-started/messaging-basics.rst [hunk 2]
```diff
@@ -3,9 +3,15 @@ Messaging Basics
 
 --------------
 
+.. |Main menu icon| image:: ../../images/sidebar_main_menu_icon.png
```
Comment (esethna) [inline source/help/getting-started/messaging-basics.rst]: Thanks @amaddio, propose we remove the icon image given it's inconsistent with the rest of our documentation, if you're okay with that?

**Comment by esethna** [comment]
Thanks @amyblais, in the future can we please make sure to add the "Editor Review" label to PRs to catch things such as the inconsistencies notes above (ie the menu icon and the formatting of the setting breadcrumbs)?

File: source/help/getting-started/messaging-basics.rst (hunk 2 above)
Comment (amaddio) [reply source/help/getting-started/messaging-basics.rst]: You are right. It is called `Main Menu` on other pages and also defined e.g. [account settings page](https://docs.mattermost.com/help/settings/account-settings.html?highlight=main%20menu#send-messages-on-ctrl-enter). So let's get rid of the icon.

File: source/help/getting-started/messaging-basics.rst (hunk 1 above)
Comment (amaddio) [reply source/help/getting-started/messaging-basics.rst]: Sounds good to me 👍

**Comment by amyblais** [comment]
Hi @amaddio, the changes suggested above by ``esethna`` will also need to be edited on this pull request. Let me know if you have any questions.

**Comment by amyblais** [comment]
@esethna amaddio has applied the changes you requested, can you help give this a second review?

**Pull request merged**